paper-drawer-panel.html

```diff
@@ -230,6 +230,8 @@
       }
 
       .left-drawer.narrow-layout > #drawer:not(.iron-selected) {
+        visibility: hidden;
+
```

Comment:
This is ok since the parent `<paper-drawer-panel>` is what captures the track events.